src/Symfony/Component/Messenger/EventListener/SendFailedMessageForRetryListener.php

```diff
@@ -43,7 +45,9 @@ public function __construct(ContainerInterface $sendersLocator, ContainerInterfa
         $this->sendersLocator = $sendersLocator;
         $this->retryStrategyLocator = $retryStrategyLocator;
         $this->logger = $logger;
-        $this->eventDispatcher = $eventDispatcher;
+        $this->eventDispatcher = class_exists(Event::class) && $eventDispatcher instanceof SymfonyEventDispatcherInterface
```

Comment:
@derrabus I think you're missing an import for the `Event` class.

Reply:
Fixed, thanks.